src/mimemail.erl

```diff
@@ -1950,6 +1960,10 @@ rfc2047_decode_test_() ->
 	].
 
 encoding_test_() ->
+	Getmail = fun(File) ->
+		{ok, Email} = file:read_file(string:concat("test/fixtures/", File)),
```

Comment:
please use `filename:join` instead of `string:concat`

Reply:
fixed